Resources/public/dist/components/news/form/form.html

```diff
@@ -3,8 +3,32 @@
         <label for="news-title"><%= translations.title %></label>
         <input type="text" id="news-title" name="title" class="form-element input-bold required" data-mapper-property="title" data-validation-required="true"/>
     </div>
-    <div class="form-group">
+    <div class="form-group grid-row">
         <label for="news-content"><%= translations.content %></label>
         <textarea id="news-content" name="content" class="form-element" data-mapper-property="content"></textarea>
     </div>
+    <div class="form-group grid-row">
+        <label for="news-release-date"><%= translations.releaseDate %></label>
+        <div id="news-release-date"
```

Comment:
this container and the container `news-media` contains the definition to start a specific sulu-component. this components (date-picker and media-selection) abstracts the complete UI from this form.